react/Labs/ExperimentalModal/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import MuiCozyTheme from '../../MuiCozyTheme/'
+
+import Dialog from '@material-ui/core/Dialog'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import DialogActions from '@material-ui/core/DialogActions'
+
+import withMobileDialog from '@material-ui/core/withMobileDialog'
+
+import AppTitle from '../../AppTitle'
+
+export const ExperimentalModal = props => {
+  const { dismissAction, fullScreen, children, open } = props
+  return (
+    <MuiCozyTheme>
```

Comment:
Actually I don't think the component should come with a MuiCozyTheme wrapper. That wrapped is supposed to be provided once, at a higher level in the app. Not including it here also means that the modal's styles can be overriden by a different Theme provider, for example the inverted theme.

Reply:
Fair enough!